services/venice-controller/src/main/java/com/linkedin/venice/controller/VeniceHelixAdmin.java

```diff
@@ -2438,6 +2439,26 @@ private void constructViewResources(Properties params, Store store, int version)
           kafkaTopicConfigs.getBoolean(LOG_COMPACTION_ENABLED),
           kafkaTopicConfigs.getOptionalInt(KAFKA_MIN_IN_SYNC_REPLICAS),
           kafkaTopicConfigs.getBoolean(USE_FAST_KAFKA_OPERATION_TIMEOUT));
+      if (topicNameAndConfigs.getKey().endsWith(MaterializedView.MATERIALIZED_VIEW_TOPIC_SUFFIX)) {
+        // Send SOP CM to materialized view topic with sorted flag equal to false due to reshuffle
+        VeniceWriterOptions.Builder vwOptionsBuilder =
+            new VeniceWriterOptions.Builder(materializedViewTopicName).setUseKafkaKeySerializer(true)
+                .setPartitionCount(kafkaTopicConfigs.getInt(PARTITION_COUNT));
+        ByteBuffer compressionDictBuffer = null;
+        if (compressionDictionary != null) {
+          compressionDictBuffer = ByteBuffer.wrap(EncodingUtils.base64DecodeFromString(compressionDictionary));
+        } else if (version.getCompressionStrategy().equals(CompressionStrategy.ZSTD_WITH_DICT)) {
+          compressionDictBuffer = emptyPushZSTDDictionary.get();
+        }
+        try (VeniceWriter veniceWriter = getVeniceWriterFactory().createVeniceWriter(vwOptionsBuilder.build())) {
+          veniceWriter.broadcastStartOfPush(
+              false,
+              version.isChunkingEnabled(),
```

Comment:
We need to make sure `AASIT` and `LFSIT` would honor the store-level config: compression and chunking.